recipes-bsp/u-boot/u-boot-imx-common_2024.04.inc

```diff
@@ -7,6 +7,7 @@ SRC_URI = "git://github.com/nxp-imx/uboot-imx.git;protocol=https;branch=${SRCBRA
 SRCBRANCH = "lf_v2024.04"
 LOCALVERSION ?= "-imx_v2024.04_6.6.52-2.2.0"
 SRCREV = "6c4545203d123c246c5d7995f2893959506d28e0"
+CVE_PRODUCT = "nxp:uboot_secondary_program_loader denx:u-boot"
```

Comment:
Not sure whether this should be overridden when using the fslc bootloader

Reply:
`CVE_PRODUCT = "denx:u-boot"` -> for u-boot-fslc
`CVE_PRODUCT = "nxp:uboot_secondary_program_loader denx:u-boot"` -> for u-boot-imx
IMO, just add those variables to both inc files.

Reply:
> CVE_PRODUCT = "denx:u-boot" -> for u-boot-fslc

fslc doesn't have a CVE vendor identifier, does it? 🤔